CanDIG/CanDIGv2
# katsu update

- bump katsu version to 2.0.0
- update katsu_service to part_31
- can update to python 3.11 and alpine 3.17 by changing .env VENV_PYTHON and ALPINE_VERSION

How to test:
- delete katsu and db container, images and volumes (make version changes in .env if you want)
- go to cd/katsu/katsu_service and checkout [sonchau/part_31](https://github.com/CanDIG/katsu/pull/114)
- make build-katsu and make compose-katsu
- make test-integration, all should pass
